daffodil-core/src/main/scala/org/apache/daffodil/dsom/RuntimePropertyMixins.scala

```diff
@@ -186,6 +186,9 @@ trait DelimitedRuntimeValuedPropertiesMixin
     val qn = this.qNameForProperty("initiator")
     val typeIfStaticallyKnown = NodeInfo.String
     val typeIfRuntimeKnown = NodeInfo.NonEmptyString
+    if (initiatorRaw.value == "%") {
+      SDE("Invalid dfdl:initiator. \"%\" cannot be a dfdl:initiator.")
```

Comment:
Suggest improve message not to just say it's wrong, but suggest the user meant "%%", and remind them that % introduces DFDL entities and so must be self-escaped if you want to literally express a single % character.